bigbluebutton-html5/imports/ui/components/actions-bar/emoji-menu/component.jsx

```diff
@@ -25,19 +25,19 @@ const intlMessages = defineMessages({
     id: 'app.actionsBar.emojiMenu.awayDesc',
     description: 'adds context to away option',
   },
-  raiseLabel: {
+  raiseHandLabel: {
     id: 'app.actionsBar.emojiMenu.raiseLabel',
     description: 'raise hand emoji label',
   },
-  raiseDesc: {
+  raiseHandDesc: {
     id: 'app.actionsBar.emojiMenu.raiseDesc',
     description: 'adds context to raise hand option',
   },
-  undecidedLabel: {
+  neutralLabel: {
```

Comment:
I would leave `undecidedLabel`, since it's "undecided" everywhere else, including the name of the icon.

Reply:
@OZhurbenko  this name was putted for follow the name of the emojis in server.